components/SearchResults/index.js

```diff
@@ -37,12 +37,12 @@ function SearchResults({
         />
       )}
       {hasEntries && (
-        <StyledSearchButton
+        <StyledCenteredButton
           type="button"
           onClick={() => handleSearchDictionaryButtonSubmit()}
```

Comment:
It can just be `onClick={handleSearchDictionaryButtonSubmit}`

Reply:
This did not work without the brackets either 😐